src/components/app-progress-bar-line/AppProgressBarLine.tsx

```diff
@@ -7,23 +7,43 @@ import { styles } from '~/components/app-progress-bar-line/AppProgressBarLine.st
 import { LinearProgress } from '@mui/material'
 import { UserRoleEnum } from '~/types'
 
+import { AccessTime } from '@mui/icons-material'
 interface AppProgressBarLineProps {
   value: number
   userRole: UserRoleEnum | ''
+  isCooperationActivities?: boolean
 }
 
 const AppProgressBarLine: FC<AppProgressBarLineProps> = ({
   value,
-  userRole
+  userRole,
+  isCooperationActivities = false
 }) => {
   const { isMobile } = useBreakpoints()
   const labelsValue =
     userRole === UserRoleEnum.Student
       ? [0, 25, 50, 75, 100]
       : [0, 20, 40, 60, 80, 100]
-
-  const labelsWithPercent = isMobile ? (
-    <Typography color={'primary.500'} variant='subtitle2'>
+  const labelsWithPercentForCooperation = (
+    <Box width={'100%'}>
+      <Typography sx={styles.primaryLabelsCoop} variant='body2'>
+        Your progress
```

Comment:
You need to translate this into both English and Ukrainian, then use the translated values with `react-i18next`